cli/policy.go

```diff
@@ -365,9 +339,12 @@ func importPolicy(ctx *cli.Context) {
 			return
 		}
 
-		if err := client.PolicyAdd(node.Name, node); err != nil {
+		jsonPolicy := node.JSONMarshal()
+		if resp, err := client.PolicyPut(node.Name, jsonPolicy); err != nil {
 			fmt.Fprintf(os.Stderr, "Could not import policy directory %s: %s\n", path, err)
 			os.Exit(1)
+		} else if ctx.Bool("echo") {
```

Comment:
The feedback is a zero or non-zero exit status plus verbose error information on stderr in case of the latter. If a user wants to have output on stdout on success he can use `--echo`